TESTS/psa/entropy_inject/main.cpp

```diff
@@ -16,7 +16,7 @@
 * limitations under the License.
 */
 
-#if ((!defined(TARGET_PSA) || (!defined(COMPONENT_PSA_SRV_IPC)) && !defined(MBEDTLS_ENTROPY_NV_SEED)))
+#if (!defined(TARGET_PSA) || !defined(COMPONENT_PSA_SRV_IPC))
```

Comment:
Where do we want the entropy injection tests to run? I'm not sure these defines are right if we want it to run on both "K64F when MBEDTLS_ENTROPY_NV_SEED is configured" and on a "PSoC 6's NSPE when the PSoC 6's SPE has MBEDTLS_ENTROPY_NV_SEED configured".

Reply:
@Patater ideally the test should run on dual-MCU(PSoC6)  or v8-m systems
this is being taken care of by `COMPONENT_PSA_SRV_IPC` define which the K64F will never have

Reply:
We probably want this test to keep working on K64F or other single v7-M targets.